src/pages/api/phd-fellowship/application.ts

```diff
@@ -51,11 +51,49 @@ const registrationHandler = async (
     files: { uploads: FormidableFile };
   };
 
+  let applicantFolderId;
+  const date = new Date();
+
+  if (files?.uploads) {
```

Comment:
lets move `date` initialization to the if statement block if its only being used there

Reply:
Actually it is used later on (see line 166).

Reply:
thats what i get for not viewing the full file, thanks for clarifying